x/mongo/driver/batch_cursor.go

```diff
@@ -334,6 +339,9 @@ func (bc *BatchCursor) getMore(ctx context.Context) {
 			if bc.maxTimeMS > 0 {
 				dst = bsoncore.AppendInt64Element(dst, "maxTimeMS", bc.maxTimeMS)
 			}
+			if bc.comment.Type != bsontype.Type(0) && bc.serverDescription.WireVersion.Max >= 9 {
```

Comment:
Can you add a comment above this line explaining why we have a wire version check?